frontend/src/pages/chat/components/chat-sidebar/styles.module.scss

```diff
@@ -2,6 +2,7 @@
   display: flex;
   justify-content: center;
   align-items: center;
+  max-height: 30px;
```

Comment:
remove this everywhere, this should be done without it
```suggestion
```

Reply:
we should consider case where text will take two lines, and header height will be different in this case

Reply:
Ok, understood. I just thought it's ok to have less paddings in case when header has two rows